core/storage/suggestion/gae_models.py

```diff
@@ -199,11 +204,11 @@ def get_export_policy(cls):
             'target_version_at_submission':
                 base_models.EXPORT_POLICY.EXPORTED,
             'status': base_models.EXPORT_POLICY.EXPORTED,
-            'author_id': base_models.EXPORT_POLICY.EXPORTED,
-            'final_reviewer_id': base_models.EXPORT_POLICY.EXPORTED,
+            'author_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'final_reviewer_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
             'change_cmd': base_models.EXPORT_POLICY.EXPORTED,
-            'score_category': base_models.EXPORT_POLICY.EXPORTED,
-            'language_code': base_models.EXPORT_POLICY.EXPORTED
+            'score_category': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
Why change this and the next value?

Reply:
These were incorrect before, they are in fact exported. As part of this testing, I discovered some errors with the labels of fields.

Reply:
I'm not sure I understand your comment. If they're being exported, the export policy shouldn't be "not applicable", right? But I think they're not being exported because they are either user IDs or the user IDs of other people?

May be worth adding a clear comment above each of the relevant lines (author_id, final_reviewer_id) to explain exactly what's going on here.